src/aat/java/uk/gov/hmcts/reform/em/hrs/DownloadScenarios.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.em.hrs;
+
+
+import io.restassured.RestAssured;
+import org.junit.Test;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+public class DownloadScenarios {
+
+    @Test
+    public void testGetHearingRecording() {
+            RestAssured
+                .given()
+                .baseUri("http://localhost:8080")
+                .contentType(APPLICATION_JSON_VALUE)
+                .get("/documents/hearing-recordings/TEST-REF/segments/TEST-SEGMENT")
```

Comment:
how will TEST-REF and TEST-SEGMENT get populated?

Reply:
To my understanding, this is the link we’re sending to get a download request, the link will identify the file and then ref and segment will come from that file which has been identified